scripts/bump-version.sh

```diff
@@ -21,41 +22,41 @@ if [[ $LAST_COMMIT_MESSAGE == *"[USE_CUSTOM_SEMVER]"* ]]; then
 else
     CURRENT_VERSION="$(npm view ${PACKAGE_NAME}@latest version)"
     echo "CURRENT_VERSION is $CURRENT_VERSION"
-
+    
```

Comment:
I'm wondering if you have the [editorconfig](https://editorconfig.org/) plugin installed in your code Editor. Here it's trivial, but if someone else want to edit this file in the future, all theses space chars might be automatically removed, this is good but this will also spoils the git history and the future PR for this file. So it might be better you remove them first (with editorconfig, your IDE will do it automatically).

Reply:
I've made some commits which should address these issues. Check it out and let me know! 😄